draft-packages/guidance-block/KaizenDraft/GuidanceBlock/GuidanceBlock.scss

```diff
@@ -35,82 +35,88 @@ $ca-banner-collapse-height-delayed: margin-top $ca-duration-fast
 
   @include ca-media-desktop {
     min-height: $heightConstraint;
+    box-sizing: border-box;
```

Comment:
Hopefully this doesn't cause any issues, but I want to document this just in case, we recently had an issue pop up on performance-ui around box-sizing and a global style that performance has: https://github.com/cultureamp/kaizen-design-system/pull/1531